bindings/python/src/decoders.rs

```diff
@@ -678,6 +678,19 @@ impl PyDecodeStream {
             prefix_index: 0,
         }
     }
+
+    /// Streaming decode step
+    ///
+    /// Args:
+    ///     tokenizer (:class:`~tokenizers.Tokenizer`):
+    ///        The tokenizer to use for decoding
+    ///    id (:obj:`int` or `List[int]`):
+    ///       The next token id or list of token ids to add to the stream
+    /// 
```

Comment:
Fixes the failing GitHub Actions test.
```suggestion
    ///
```